### PR TITLE
Make instructions clearer for patching single library

### PR DESCRIPTION
**Pull request by StephenGoodall**
https://github.com/aws/aws-xray-sdk-python/issues/71

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: doc-source/xray-sdk-python-patching.md [hunk 1]
```diff
@@ -54,6 +54,38 @@ In some cases, the key that you use to patch a library does not match the librar
...
+import requests
+import mysql-connector-python
+
+from aws_xray_sdk.core import xray_recorder
+from aws_xray_sdk.core import patch
+
+libraries = (['botocore'])
+patch(libraries)
+```
+
```
Comment (willarmiros) [inline doc-source/xray-sdk-python-patching.md]: Please remove this example from the PR. We typically only have 1 recommended way of doing something in the documentation, and the other example is more consistent with patching calls documented above.

File: doc-source/xray-sdk-python-patching.md [hunk 2]
```diff
@@ -54,6 +54,38 @@ In some cases, the key that you use to patch a library does not match the librar
 `httplib` – `[httplib](https://docs.python.org/2/library/httplib.html)` and [https://docs.python.org/3/library/http.client.html](https://docs.python.org/3/library/http.client.html)
 `mysql` – `[mysql\-connector\-python](https://pypi.python.org/pypi/mysql-connector-python)`
 
+To patch a single library, call `patch` with a tuple of the library name\. In order to achieve this you will need to add a trailing comma or provide a single element list\.
+
+**Example main\.py – patch single library (trailing comma)**
```
Comment (willarmiros) [inline doc-source/xray-sdk-python-patching.md]: "Trailing comma" specification no longer needed in title here.

**Review by willarmiros — CHANGES_REQUESTED** [review]
Hi @StephenGoodall,
Thank you for the contribution. I've left some suggestions in the comments.

**Comment by StephenGoodall** [comment]
Hi @willarmiros I've made those changes 😃 
Let me know if it needs anything else!

**Review by willarmiros — APPROVED** [review]

**Comment by alivest** [comment]
Thank you Stephen!

**Pull request merged**